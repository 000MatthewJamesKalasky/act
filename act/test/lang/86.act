defproc testme (chan?(int) a; chan!(int) b)
{
  chan(int) w;
  dataflow { 
    a{2..0} -> b;
    b -> w
  }
}

testme t;
