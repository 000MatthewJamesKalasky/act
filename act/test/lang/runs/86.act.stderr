ERROR: File `86.act', line 6, col 7
	Typechecking failed on expression!
	Channel expression requires an input port
